modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -73,8 +73,18 @@ You can install {product-title} version {product-version} on the following IBM h
 [discrete]
 === Hardware requirements
 
-* 1 LPAR with 6 IFLs that supports SMT2
-* 1 OSA or RoCE network adapter
+* The equivalent of 6 IFLs, which are SMT2 enabled, for each cluster.
+* At least one network connection to connect to the LoadBalancer service and for traffic outside the cluster.
```

Comment:
Most of the references I see in the collection are either to specific services named `LoadBalancer` or references in plain text to a "load balancer." Please confirm this usage.

Also, is this one network connection? If so, consider something like:
```suggestion
* At least one network connection to both connect to the LoadBalancer service and serve data to traffic outside the cluster
```